apps/developer-portal/src/components/identity-providers/identity-provider-list.tsx

```diff
@@ -103,7 +123,7 @@ export const IdentityProviderList: FunctionComponent<IdentityProviderListPropsIn
      * @param {string} idpId Identity provider id.
      */
     const handleIdentityProviderEdit = (idpId: string): void => {
-        history.push(`identity-providers/${ idpId }`);
+        history.push(ApplicationConstants.PATHS.get("IDP_EDIT").replace(":id", idpId));
```

Comment:
Should we add this to IdentityProviderConstants?

Reply:
I was thinking of adding all the app paths to a map in `ApplicationConstants`. Maybe the name of the file is misleading and can be named as `AppConstants` if needed. `Routes ` config also should use the paths from this file rather than hardcoding.